jetty-integrations/jetty-openid/src/test/java/org/eclipse/jetty/security/openid/OpenIdRealmNameTest.java

```diff
@@ -74,8 +76,10 @@ public void testSingleConfiguration() throws Exception
     public void testSingleConfigurationNoRealmName() throws Exception
     {
         // Add some OpenID configurations.
-        OpenIdConfiguration config1 = new OpenIdConfiguration("provider1",
-            "", "", "", "", null);
+        OpenIdConfiguration config1 = new OpenIdConfiguration.Builder("provider1", "", "")
+            .authorizationEndpoint("")
+            .tokenEndpoint("")
```

Comment:
If there is a semantic difference between `""`, `null` and not set at all, then you need to say so in the javadoc

Reply:
A value of `null` means it hasn't been set. And it already says in the javadoc if this has not been set it will search for the OpenID metadata at `.well-known/openid-configuration`.

The authorizationEndpoint/tokenEndpoint value of `""` has no special meaning, and will not actually work. In this test I just need a non-null value.

So there is no point mentioning `""` specifically in the javadoc because it means nothing and is just an incorrect value.

Reply:
Could you set it to "test" instead? That avoids the confusion if "" is unsetting it or not

Reply:
done